packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -190,3 +206,116 @@ export function getNetworkName(chainId: number) {
       return 'Unknown'
   }
 }
+
+export function getNetworkLogo(chainId: number) {
+  switch (chainId) {
+    // L1 networks
+    case ChainId.Mainnet:
+      return EthereumLogo
+
+    case ChainId.Rinkeby:
+      return EthereumLogo
+
+    case ChainId.Goerli:
+      return EthereumLogo
+
+    // L2 networks
+    case ChainId.ArbitrumOne:
+      return ArbitrumOneLogo
+
+    case ChainId.ArbitrumNova:
+      return ArbitrumNovaLogo
+
+    case ChainId.ArbitrumRinkeby:
+      return ArbitrumOneLogo
+
+    case ChainId.ArbitrumGoerli:
+      return ArbitrumOneLogo
```

Comment:
```suggestion
    case ChainId.ArbitrumNova:
      return ArbitrumNovaLogo

    case ChainId.ArbitrumOne:
    case ChainId.ArbitrumRinkeby:
    case ChainId.ArbitrumGoerli:
      return ArbitrumOneLogo
```

also this?
we can move default to the previous block but I think it's also okay to keep it separate for clarity